CITATION.cff

```diff
@@ -12,5 +12,9 @@ authors:
 - family-names: "Schmiedmayer"
   given-names: "Paul"
   orcid: "https://orcid.org/0000-0002-8607-9148"
+- family-names: "Ravi"
+  given-names: "Vishnu"
+  orcid: "https://orcid.org/0000-0003-0359-1275"
```

Comment:
It would rather make sense to add you and Adrit here for this repo as he was also part of the Spezi Speech PRs in SpeziML.